test/src/unittests/rhythm/test_tempotapdegara.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
```

Comment:
Double check if this is a duplicate of test_tempotap.
This should be a dummy test.

Reply:
This file will be removed. It is wrongly based on test_tempotap and needs to be re-written from scratch